src/react-apps/applications/ux-editor/src/sagas/formFiller/formFillerSagas.ts

```diff
@@ -119,3 +119,25 @@ export function* resetFormDataSaga({ url }: FormFillerActions.IResetFormDataActi
 export function* watchResetFormDataSaga(): SagaIterator {
   yield takeLatest(FormFillerActionTypes.RESET_FORM_DATA, resetFormDataSaga);
 }
+
+export function* runSingleFieldValidationSaga({
+  url,
+  dataModelBinding,
+}: FormFillerActions.IRunSingleFieldValidationAction): SagaIterator {
+  const state: IAppState = yield select();
+  try {
+    const response = yield call(put, url, 'Validate',
+    convertDataBindingToModel(state.formFiller.formData, state.appData.dataModel.model), null, dataModelBinding);
+    if (response && response && (response.status === 1 || response.status === 2)) {
```

Comment:
Is it necessary to check response twice?

Reply:
😆 
No, that seems a bit extreme. Updated.